### ISSUE TITLE
Transaction Number no longer accepts plain text

### ISSUE THREAD

**Issue by Gigific** [moneymanagerex/moneymanagerex#4975]
MMEX version:
 - [ ] 1.5.17
 - [ ] 1.5.18
 - [X] 1.5.19
 - [ ] 1.5.20 Beta
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Transaction number was treated as plain text before and allowed to enter a number in alphanumeric form (i.e RT123). Since version 1.5.19 it's treated as a numeric field: it allows to perform math calculations as in the "amount" field, but eliminates any character that is not a number from it.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
Write an alphanumeric string in the "number" field:
![immagine](https://user-images.githubusercontent.com/110192669/186749462-fe6e4aae-19f3-4037-9502-1b852dbccf4c.png)
Click on another field and see that the all the letters disappear:
![immagine](https://user-images.githubusercontent.com/110192669/186749509-cd0bfd04-96da-4468-9a8c-a9ef28b499b1.png)

Expected result:
Even if the definition of "number" is pretty straightforward, a transaction number may also include alphanumeric characters and this possibility should not be inhibited, as in all previous releases.

Actual result:
Any non-number character is removed from the field.

**Comment by whalley** [comment]
Thanks for reporting. Will be fixed in v1.5.20.
Closing as duplicate of https://github.com/moneymanagerex/moneymanagerex/issues/4951

**Comment by tactilis** [comment]
@Gigific
Revert to 1.5.18 until 1.5.20 is out if this bug is causing you problems
